server/graphql/lib.rs

```diff
@@ -118,10 +118,6 @@ impl CentralServerQueryNode {
     async fn plugin(&self) -> CentralPluginQueries {
         CentralPluginQueries
     }
-
-    async fn campaign(&self) -> CampaignQueries {
-        CampaignQueries
-    }
```

Comment:
Removed the `campaign` query from the central server collection of queries. I didn’t remove the `campaign` mutations under `central server`, as I think we’d likely only ever create campaigns there?

Support for inserting and removing campaigns on `Stock` lines and `Inbound Shipment` lines is already in place outside the `central server` mutations. We just needed a way to query the campaigns outside of central server

Reply:
Yup exactly